### ISSUE TITLE
Use embedded playground to show example in tutorial of HTML5 IndexDB

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#464]
```
Hi Paul,
I'd like to invite you to try embedded playground in the your tutorial of the 
"A Simple TODO list using HTML5 IndexedDB", esp. the example in "The final 
product" part.

The tutorial and help doc links are listed below:

Tutorial Link:
http://www.html5rocks.com/tutorials/indexeddb/todo/

Guide on how to use embedded playground:
https://docs.google.com/a/google.com/document/d/1ReJCAeh9wW-JO07Wx7uWferqg3BQwMP
dubORobNkQwU/edit?hl=en#

Feel free to let me know if you have any questions.

Thanks!

-hanrui
```

Original issue reported on code.google.com by `han...@google.com` on 25 Feb 2011 at 7:11

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 11 Apr 2011 at 4:36
- Added labels: **Milestone-6**
- Removed labels: **Milestone-5.4**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 18 Aug 2011 at 6:04
- Added labels: **Milestone-7.1**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Revision 5b34aafd9146.
```

Original comment by `han...@google.com` on 20 Sep 2011 at 11:40
- Changed state: **Fixed**
- Added labels: **Milestone-7.2**
- Removed labels: **Milestone-7.1**
